lib/blacklight/models/module_item.rb

```diff
@@ -8,7 +10,7 @@ def initialize(title, type, identifierref)
       @workflow_state = "active"
     end
 
-    def canvas_conversion
+    def canvas_conversion(*)
```

Comment:
yeah, same as doing `def meh(*args)` but in this case I didn't care about the variable as it isn't used.